metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -130,13 +133,33 @@ object BuildServerConnection {
     )
   }
 
+  final case class BloopExtraBuildParams(
+      semanticdbVersion: String,
+      supportedScalaVersions: Seq[String]
```

Comment:
make it a java.List and you probably won't need a `toJson` method - and it will match the other structures in the bsp/lsp protocol.

Reply:
Done, thanks for the idea!